sources/engine/Stride/Rendering/ParameterCollection.cs

```diff
@@ -21,6 +21,8 @@ namespace Stride.Rendering
     [DebuggerTypeProxy(typeof(ParameterCollection.DebugView))]
     public class ParameterCollection
     {
+        private const int Alignment = 16;
```

Comment:
Can you add a comment on this line `// Important! Alignment must be a power of 2` just in case?